### PR TITLE
Code clean up

### PR DESCRIPTION
**Pull request by worldofnick**
Hey guys, first time contributing! I went though some code and tried to find any places things could be cleaned up. This doesn't address any specific bug, but I thought this might be slightly useful and a good way to see some of the code.

I'm hoping someone with more knowledge of repo can take a look at commit `e0292db`, seemed a bit strange to me.

Also, are performance PR's wanted? I thought I may have seen some places to improve some performance and if so I'd love to help out.

If this satisfies any bounty my XRB address is `xrb_15fy38cbw7t4tmwwrg3dri1w9r4hnabwcam787asoqb53a1mxe8mfqipahu8`

### CODE REVIEW

**Comment by lukealonso** [comment]
Please fix the various indentation issues (use tabs), and squash your commits into one.

File: rai/node/wallet.cpp [hunk 1]
```diff
@@ -1264,7 +1264,6 @@ std::shared_ptr<rai::wallet> rai::wallets::create (rai::uint256_union const & id
 		rai::transaction transaction (node.store.environment, nullptr, true);
 		result = std::make_shared<rai::wallet> (error, transaction, node, id_a.to_string ());
 		items[id_a] = result;
-		result = result;
```
Comment (lukealonso) [inline rai/node/wallet.cpp]: Weird, I can't figure out what was really intended here.

**Comment by worldofnick** [comment]
@lukealonso thanks for pointing out the indents, Xcode was acting strange.

**Comment by worldofnick** [comment]
I don't think I can squash the commits, 

> To squash and merge pull requests, you must have write permissions in the repository, and the repository must allow squash merging.

I don't have write permissions, I think whoever merges this will need to `Squash & Merge`.

**Comment by cryptocode** [comment]
@worldofnick you squash locally on your branch, then you push it with --force.

**Pull request merged**